### DISCUSSION TITLE
baltic dry index

### DISCUSSION THREAD

**Discussion by ogipy** [alvarobartt/investpy#577] [Q&A]
I wanna get "baltic dry index" by using investpy. 
I tried investpy.get_index_historical_data(index='baltic dry index', country='united states', from_date='01/01/2018', to_date='01/01/2019')
but error occurred.
Please help me.

**Comment by albcastano** [comment]
Try with:

search_result = investpy.search_quotes(text="BADI", n_results=1)
historical_data = search_result.retrieve_historical_data(from_date='01/01/2018',to_date='01/01/2019')

**Reply by ogipy** [reply]
Thank you for your answer.
I could get what I'm looking for.
Thank you very much!
